pnpm-lock.yaml

```diff
@@ -10311,7 +10311,7 @@ packages:
     resolution: {integrity: sha512-RAmZReFuKCKx0Rs5C0nVJwKomAHUHn+gGP/YvbEsXQWu0sXoncEUZa71UqlfCPVXa/0MkYOIbCXSQdOcuRrHgw==}
     peerDependencies:
       graphql: ^14.0.0 || ^15.0.0
-      react: ^16.8.0 || ^17.0.0
+      react: ^16.8.0 || ^17.0.0 || 17.0.2
```

Comment:
this change is interesting. I wonder why it happened.

Reply:
Because we have a peer dep in package.json:
	"pnpm": {
		"overrides": {
			"@types/node": "^14.18.36"
		},
		"peerDependencyRules": {
			"allowedVersions": {
				"react": "17.0.2",
				"react-dom": "17.0.2",
				"webpack": "5.75"
			},
			"ignoreMissing": [
				"fluid-framework"
			]
		}
	}

Reply:
I have tried and failed to figure that out. It repros for me and is probably fine